<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Login Logout"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="25.289">
  <properties />
            <testcase name="createNewCustomerWithAllDetails" time="25.289" classname="com.mobizio.testscript.CreateNewCustomerWithAllDetails">
                <failure type="java.lang.AssertionError"
                  message="expected [Automationdhrf] but found [Automationhpbq]"
              >
<![CDATA[java.lang.AssertionError: expected [Automationdhrf] but found [Automationhpbq]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  org.testng.Assert.assertEquals(Assert.java:186)
  com.mobizio.pages.CustomerPage.verifyCreatedCustomerOnCustomerPage(CustomerPage.java:282)
  com.mobizio.testscript.CreateNewCustomerWithAllDetails.createNewCustomerWithAllDetails(CreateNewCustomerWithAllDetails.java:88)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
